integrations/access/servicenow/app.go

```diff
@@ -436,11 +436,13 @@ func (a *App) tryApproveRequest(ctx context.Context, req types.AccessRequest) er
 }
 
 func (a *App) getOnCallUsers(ctx context.Context, serviceNames []string) ([]string, error) {
+	log := logger.Get(ctx)
 	onCallUsers := []string{}
 	for _, scheduleName := range serviceNames {
 		respondersResult, err := a.serviceNow.GetOnCall(ctx, scheduleName)
 		if err != nil {
-			return nil, trace.Wrap(err)
+			log.WithError(err).Error("Failed to retrieve responder from schedule")
+			continue
```

Comment:
Should we distinguish `not found` errors from other types of errors?
Also, can you please add test coverage for this change?

Reply:
Is this along the lines of what you were thinking?

Reply:
Yes, thank you.